renovate.json

```diff
@@ -1,10 +1,6 @@
 {
-  "extends": [":library"],
+  "extends": [":app"],
```

Comment:
Renovate deduced that you were using a non-public package.json (i.e. you haven't set `private=true`) so therefore defaults to the `:library` preset. `:library` is essentially the same as `:app` except that it retains semver ranges for `dependencies`, because it's considered good form for libraries to not pin dependencies and risk causing duplicates in downstream packages.

Anyway I have changed this to `:app` which pins everything, so there's then no need for `depTypes` version pinning added manually now.